app.py

```diff
@@ -76,44 +75,6 @@ def proposals():
     return jsonify({'proposals': service_proposals})
 
 
-# TODO: make 'session_id' param required after client update
-@app.route('/v1/client_create_session', methods=['POST'])
-@validate_json
-def client_create_session():
-    payload = request.get_json(force=True)
-    node_key = payload.get('node_key', '')
-
-    node = Node.query.get(node_key)
-    if not node:
-        return jsonify(error='node key not found'), 400
-
-    if node.get_status() != 'active':
-        return jsonify(error='node is not active'), 400
-
-    try:
-        obj = json.loads(node.connection_config)
-    except:
-        return jsonify(error='cannot deserialize service proposal'), 400
-
-    service_proposal = obj.get('service_proposal', None)
-    if service_proposal is None:
-        return jsonify(error='service_proposal was not found'), 400
-
-    session_key = payload.get('session_key') or helpers.generate_random_string()
-    session = Session(session_key)
-    session.node_key = node_key
-    session.client_updated_at = datetime.utcnow()
```

Comment:
We need new sessionSaving endpoint, with same features

Reply:
We decided to save session in stats endpoints, not on a separate endpoint.
On stats endpoint, new session will be created with `session_id` if it does not exist already.

Reply:
Just make a note and dont forget fields:
- `client_updated_at`
- `client_ip`

Reply:
Added a note here: https://mysteriumnetwork.atlassian.net/browse/MYST-142